rotary/rules/polish.smk

```diff
@@ -55,7 +53,7 @@ rule prepare_polypolish_polish_input:
     input:
         "{sample}/polish/medaka/{sample}_consensus.fasta"
     output:
-        temp("{sample}/polish/polypolish/input/{sample}_input.fasta")
+        "{sample}/polish/polypolish/input/{sample}_input.fasta"
```

Comment:
@jmtsuji Was removing `temp()` on purpose here?

Reply:
Thanks for bringing this up. The corresponding rule for the circularize module, `prepare_polypolish_circularize_input`, did not have `temp()` on its output file, so there was some inconsistency between the two rules. To address this, I decided to play things on the safe side and set the output of both `prepare_polypolish_circularize_input` and this rule to non-temp. I could try setting both to `temp()` instead but will need to test if that affects the pipeline running correctly as in #132 -- thoughts?

Reply:
@jmtsuji 

> I could try setting both to temp() instead but will need to test if that affects the pipeline running correctly as in https://github.com/rotary-genomics/rotary/issues/132 -- thoughts?

Let's try this.

Reply:
Tests are done -- after changing the output of both rules to temp(), the behaviour seems identical to what was found for #132, so I think it's okay to set both as temp().